unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionSceneContributorsSettingsController.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+internal class SectionSceneContributorsSettingsController : SectionBase, ISelectSceneListener, ISectionUpdateSceneContributorsRequester
+{
+    public const string VIEW_PREFAB_PATH = "BuilderProjectsPanelMenuSections/SectionSceneContributorsSettingsView";
+    
+    public event Action<string, SceneContributorsUpdatePayload> OnRequestUpdateSceneContributors;
+
+    private readonly SectionSceneContributorsSettingsView view;
+    private readonly FriendsSearchPromptController friendsSearchPromptController;
+    private readonly UserProfileFetcher profileFetcher = new UserProfileFetcher();
+    private readonly SceneContributorsUpdatePayload contributorsUpdatePayload = new SceneContributorsUpdatePayload();
+
+    private List<string> contributorsList = new List<string>();
+    private string sceneId;
+
+    public SectionSceneContributorsSettingsController() : this(
+        Object.Instantiate(Resources.Load<SectionSceneContributorsSettingsView>(VIEW_PREFAB_PATH)),
+        FriendsController.i
+    )
+    {
+    }
+    
+    public SectionSceneContributorsSettingsController(SectionSceneContributorsSettingsView view, IFriendsController friendsController)
```

Comment:
I would create an interface for `SectionSceneContributorsSettingsView` and use it here in order to make the dependency of the view mockeable and be able to create tests for `SectionSceneContributorsSettingsController` (and not only for `SectionSceneContributorsSettingsView`).